docs/08-Fuel/Fuel-Algorithms/AlphaN.md

```diff
@@ -1 +1,62 @@
-# AlphaN Strategy
+# Alpha-N Strategy
+
+## Introduction
+Alpha-N is a fuel strategy that implies calculating engine load using the TPS (Throttle Position Sensor). Initially it was a fallback mode built into BMW engine management units used only if the MAF (Mass Air-Flow) sensor had a failure or was missing. Nowadays, it can be a suitable method for estimating load and tuning engines with certain setups, and even better if combined with other strategies we will discuss further.
+
+## How does it work?
+<img width="1546" height="630" alt="image" src="https://github.com/user-attachments/assets/a14d036c-af46-45d7-bb8d-dfef62e3c413" />
+
+In esscence, this fueling method is quite similar to Speed Density, you're equally presented with both a VE table and a target AFR table. Where X axis means RPM, Y Axis means Load and the cell value means VE (or target AFR on said table). The main difference is that the Y Axis will represent **Throttle Position** instead of MAP (Manifold Air Pressure). 
```

Comment:
```suggestion
In essence, this fueling method is quite similar to Speed Density, you're equally presented with both a VE table and a target AFR table. Where X axis means RPM, Y Axis means Load and the cell value means VE (or target AFR on said table). The main difference is that the Y Axis will represent **Throttle Position** instead of MAP (Manifold Air Pressure). 
```